sse2neon.h

```diff
@@ -2745,12 +2803,22 @@ FORCE_INLINE __m128i _mm_undefined_si128(void)
 #if defined(__GNUC__) || defined(__clang__)
 #pragma GCC diagnostic push
 #pragma GCC diagnostic ignored "-Wuninitialized"
+#endif
+#if defined(_MSC_VER)
+#pragma warning(push)
+#pragma warning(disable : 4700)
```

Comment:
This doesn't actually work, so I gave up and just used setzero below. However, it looks like it *does* work if you move the #pragma above the function declaration.

The other thing to watch out for is runtime uninitialized variable checks. If __MSVC_RUNTIME_CHECKS is defined, the setzero needs to be there or those checks will cause a crash at runtime.

Reply:
Honestly, I think I'm okay with setzero in all cases - avoids the headaches with `__MSVC_RUNTIME_CHECKS`